source/agora/consensus/data/Block.d

```diff
@@ -134,8 +134,7 @@ public struct BlockHeader
     /***************************************************************************
 
         Returns:
```

Comment:
Should be located below the `Params`, it's not been located here by this PR tho.

Reply:
Let's leave it for a different PR.